docs/configure.rst

```diff
@@ -192,6 +192,9 @@ For example, if we have set it to ``warning``, no debug or informational message
 Finally, there is a special set of handlers for handling performance log messages.
 These are stored in the ``handlers_perflog`` property.
 The performance handler in this example will create a file per test and per system/partition combination and will append the performance data to it every time the test is run.
```

Comment:
```suggestion
The performance handler in this example will create a file per test and per system/partition combination and will append the performance data to it every time the *performance test* is run.
```

Then either we define what a performance test is or have link to somewhere that we define it.